dom.bs

```diff
@@ -1201,6 +1201,9 @@ for discussion).
  <a>relatedTarget</a> against <var>target</var> if <var>event</var>'s <a>relatedTarget</a> is
  non-null, and null otherwise.
 
+ <li><p>If <var>event</var>'s {{Event/isTrusted}} attribute is true and <var>target</var> is
+ <var>relatedTarget</var>, then terminate these steps.
```

Comment:
> Also, this would affect every UI event dispatch beyond just ones inside shadow trees

UA's UI Event which happens in a document tree never meets this condition.
Could you have an example of an UA's event which happens in a document tree which meets this condition?